wondrous-app/components/Common/CSVFileDropZone/styles.tsx

```diff
@@ -0,0 +1,60 @@
+import { Typography } from '@mui/material';
+import styled from 'styled-components';
+import DocumentIcon from 'components/Icons/DocumentIcon';
+import typography from 'theme/typography';
+import palette from 'theme/palette';
+
+export const DropZone = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 40px 0;
+  border: 2px dashed ${palette.grey78};
+  border-radius: 6px;
+  cursor: pointer;
+  color: ${palette.grey78};
+  font-size: 14px;
+  font-weight: 500;
+  transition: all 0.2s ease-out;
+
+  :hover {
+    border: 2px dashed ${palette.grey850};
+    color: ${palette.grey850};
+  }
+`;
+
+export const DropZoneFileContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 12px;
+`;
+
+export const DropZoneIcon = styled(DocumentIcon)`
```

Comment:
Can you please add props `width and height` to the DocumentIcon add pass width/height in the component ?